src/pages/AddressPage/index.tsx

```diff
@@ -382,10 +382,22 @@ export function AddressPage() {
 
           {type === "erc20" &&
             <Tab title={<Text size="small">Events</Text>}>
-              <EventsTab id={id}/>
+              <EventsTab id={id} />
             </Tab>
           }
 
+          {type === "erc721" || type === "erc1155" || type === "erc20" ? (
+            <Tab title={<Text size="small">Tools</Text>}>
```

Comment:
Do we need "Tools" Tab in this PR? Tab content duplicates Holders tab

Reply:
Ah let me remove this